ingest/template/vanilla_spreadsheet_builder.py

```diff
@@ -69,11 +76,13 @@ def build(self, spreadsheet_tabs_template: SchemaTemplate):
                         worksheet.set_row(3, None, None, {'hidden': True})
 
                     if column_index == 0:
-                        worksheet.set_row(0, 30)
-                        worksheet.set_row(4, 30)
-                        worksheet.write(4, column_index, "FILL OUT INFORMATION BELOW THIS ROW", self.header_format)
+                        worksheet.set_row(0, 50)  # set row height for the user friendly name
+                        worksheet.set_row(1, 50)  # set row height for description
+                        worksheet.set_row(2, 50)  # set row height for example
+                        worksheet.set_row(4, 30)  # set row height for guide text row
+                        worksheet.write(4, column_index, "FILL OUT INFORMATION BELOW THIS ROW", self.guide_format)
```

Comment:
might be better to define variables for these and remove the comments  

or define a dictionary or config object for this